src/crypto/bcrypt/password.cr

```diff
@@ -15,7 +15,7 @@ require "../subtle"
 #
 # See `Crypto::Bcrypt` for hints to select the cost when generating hashes.
 class Crypto::Bcrypt::Password
-  SUPPORTED_VERSIONS = ["2", "2a", "2b"]
+  private SUPPORTED_VERSIONS = ["2", "2a", "2b"]
```

Comment:
While we're already in here, should we throw in `2y`? Is the version that is used in PHP based implementations.

Reply:
@Blacksmoke16 sure, I'll just do it in a separate PR since I'd also like to add tests etc., so it's not a 1 line change.